fixtures/tecshow/nebula-6.json

```diff
@@ -0,0 +1,180 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "Nebula 6",
+  "shortName": "nebula6",
+  "categories": ["Color Changer"],
+  "meta": {
+    "authors": ["Esteban Zallocco"],
+    "createDate": "2021-08-18",
+    "lastModifyDate": "2021-08-18"
+  },
+  "links": {
+    "manual": [
+      "http://amproweb.com/_admin/files/02b55bNebula%206%20B.pdf"
+    ],
+    "productPage": [
+      "http://tecshow.amproweb.com/es/nebula6"
+    ],
+    "video": [
+      "https://www.youtube.com/watch?v=IQQqpnKiArE"
+    ],
+    "other": [
+      "http://amproweb.com/_admin/files/6a5bd4Nebula6.zip"
+    ]
+  },
+  "physical": {
+    "dimensions": [260, 260, 360],
+    "weight": 1,
+    "power": 80,
+    "DMXconnector": "3-pin",
+    "bulb": {
+      "type": "LED"
+    },
+    "lens": {
+      "degreesMinMax": [40, 40]
+    }
+  },
+  "availableChannels": {
+    "Red": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Red"
+      }
+    },
+    "Green": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Green"
+      }
+    },
+    "Blue": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Blue"
+      }
+    },
+    "White": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "White"
+      }
+    },
+    "Amber": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Amber"
+      }
+    },
+    "UV": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "UV"
+      }
+    },
+    "Dimmer": {
+      "capability": {
+        "type": "Intensity"
+      }
+    },
+    "Strobe": {
+      "capabilities": [
+        {
+          "dmxRange": [0, 7],
+          "type": "NoFunction"
+        },
+        {
+          "dmxRange": [8, 99],
+          "type": "StrobeSpeed",
+          "speedStart": "0Hz",
+          "speedEnd": "100Hz"
+        },
+        {
+          "dmxRange": [100, 149],
+          "type": "StrobeSpeed",
+          "speedStart": "100Hz",
+          "speedEnd": "0Hz"
+        },
+        {
+          "dmxRange": [150, 199],
+          "type": "Effect",
+          "effectName": "strobe1"
+        },
+        {
+          "dmxRange": [200, 245],
+          "type": "Effect",
+          "effectName": "strobe2"
+        },
+        {
+          "dmxRange": [246, 255],
+          "type": "StrobeSpeed",
+          "speedStart": "200Hz",
+          "speedEnd": "200Hz"
+        }
```

Comment:
Unfortunately, the manual is quite vague here.

But a strobe speed of over 100Hz or 200Hz is highly unlikely. Is there really an increasing strobe speed between DMX values 8 and 99 and then a decreasing strobe speed between DMX values 100 and 149?

What are the two strobe effects exactly?

Is the fastest strobe speed at DMX values 246 to 245 the same as between DMX values 99 and 100?

Reply:
See https://github.com/OpenLightingProject/open-fixture-library/pull/2108#issuecomment-1004209225.